modules/social_features/social_event/modules/social_event_managers/src/Plugin/Action/SocialEventManagersSendEmail.php

```diff
@@ -122,7 +208,25 @@ public function buildPreConfigurationForm(array $form, array $values, FormStateI
   /**
    * {@inheritdoc}
    */
-  public function buildConfigurationForm(array $form, FormStateInterface $form_state) {
+  public function buildConfigurationForm(array $form, FormStateInterface $form_state): array {
+    if (!empty($this->context['selected_removed'])) {
+      $this->messenger()
+        ->addWarning($this->t('Your email will be sent to members who have opted to receive community updates and announcements'));
+      $this->messenger()
+        ->addWarning($this->formatPlural($this->context['selected_removed'],
+          "1 member won't receive this email due to their communication preferences.",
+          "@count members won't receive this email due to their communication preferences."
+        ));
```

Comment:
Victoria saw screens and demos and didn't raise any objections.
Meantime, you said 
> this messenger is used to signal to the user that something didn't go according to plan

which exact match our requirements - as a group manager I expect to send an email to all users I chosen, but some of them will not receive it ("_...something didn't go according to plan_")

Reply:
I don't think the sprint demo is generally the place for design critique since there's also a lot of other people there. I also disagree that "...something didn't go according to plan" because from an application perspective everything is working just fine.

But I guess this is something that can be discussed in follow-ups if we find it important enough.